kura/org.eclipse.kura.wire.provider/src/main/java/org/eclipse/kura/internal/wire/WireServiceImpl.java

```diff
@@ -95,18 +98,18 @@ public WireServiceImpl() {
      */
     protected synchronized void activate(final ComponentContext componentContext,
             final Map<String, Object> properties) {
-        s_logger.debug(s_message.activatingWireService());
+        logger.debug(message.activatingWireService());
         try {
             this.extractProperties(properties);
             this.trackerCustomizer = new WireComponentTrackerCustomizer(componentContext.getBundleContext(), this);
-            this.serviceTracker = new ServiceTracker<WireComponent, WireComponent>(componentContext.getBundleContext(),
-                    WireComponent.class, this.trackerCustomizer);
+            this.serviceTracker = new ServiceTracker<>(componentContext.getBundleContext(), WireComponent.class,
+                    this.trackerCustomizer);
             this.serviceTracker.open();
             this.createWires();
         } catch (final Exception exception) {
-            s_logger.error(ThrowableUtil.stackTraceAsString(exception));
+            logger.error(ThrowableUtil.stackTraceAsString(exception));
```

Comment:
`org.slf4j.Logger` does not have any specific method which only takes a single object without passing a string. Hence, I needed to pass the stack trace as represented as stack trace.